deployments/kubernetes/chart/gitwebhookproxy/templates/ingress.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: extensions/v1beta1
```

Comment:
Make the whole ingress option only if `ingress` block  is given in the values

Reply:
Updated